core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -21,10 +21,10 @@ syntax = "proto3";
 // associated with it.
 package keytransparency.v1.types;
 
-import "crypto/keyspb/keyspb.proto";
-import "crypto/sigpb/sigpb.proto";
-import "trillian.proto";
-import "trillian_map_api.proto";
+import "github.com/google/trillian/crypto/keyspb/keyspb.proto";
```

Comment:
keep the relative import scheme for consistency with the other files. 
If you're getting compile errors, `protoc` most likely needs a new `-I` import for the trillian repo